igor-web/src/main/java/com/netflix/spinnaker/igor/artifactory/model/ArtifactoryRepositoryType.java

```diff
@@ -13,9 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package com.netflix.spinnaker.igor.artifactory.model;
 
 public enum ArtifactoryRepositoryType {
-  Maven,
+  MAVEN("Maven"),
```

Comment:
👍

Reply:
@robzienert , on further thought and testing , moved the artifact extension search field to the enum as well as this will remain a constant for each repo type.

Updated Halyard to exclude the field from config too.